doc: document `next('router')` behaviour (#1131)
expressjs/expressjs.com: _includes/api/en/5x/router-METHOD.md

  res.send('commit range ' + from + '..' + to)
})
```

You can use `next` primitive to implement a flow control between different
middleware functions, based on a specific program state. Invoking `next` with

> the name of a router will cause all the remaining route callbacks on that router

Reviewer: ```suggestion
the string `'router'` will cause all the remaining route callbacks on that router
```

Author: @dougwilson, fixed it. PTAL.
